manifests/server/grant.pp

```diff
@@ -75,15 +75,8 @@
   }
 
   #
-  # Port, order of precedence: $port parameter, $connect_settings[PGPORT], $postgresql::server::port
```

Comment:
Isn't it confusing to have `$port` but not use it? In that sense I think the old logic was better: make `it Optional[Stdlib::Port]` and fall back to `connect_settings` and otherwise the server port.

Reply:
updated it